trieste/space.py

```diff
@@ -984,7 +1007,7 @@ def fix_subspace(self, tag: str, values: TensorType) -> TaggedProductSearchSpace
 
     def get_subspace_component(self, tag: str, values: TensorType) -> TensorType:
         """
-        Returns the components of ``values`` lying in a particular subspace.
+        Returns the components of ``values`` lying in a particular subspace. Supports broadcasting.
```

Comment:
maybe not worth writing "support broadcasting". Most other methods do and we dont state it there?